include/verilated.cpp

```diff
@@ -2538,19 +2538,21 @@ void Verilated::debug(int level) VL_MT_SAFE {
     }
 }
 
-const char* Verilated::catName(const char* n1, const char* n2, const char* delimiter) VL_MT_SAFE {
+const char* Verilated::catName(const char* n1, const char* n2, int scope_t, const char* delimiter) VL_MT_SAFE {
```

Comment:
```suggestion
const char* Verilated::catName(const char* n1, const char* n2, int scopet, const char* delimiter) VL_MT_SAFE {
```

Or something - we reserve _t suffixes for types.